release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -36,10 +36,132 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://git
 
 ### Runtime Updates
 
-#### New Features
-
 #### Improvements
 
+##### Support to Provide Values for Configurable Variables through TOML In-line Tables
+
+The configurable feature is improved to support TOML in-line tables through the TOML syntax.
+The values for configurable variables of types `map` and `record` can be now provided using TOML in-line tables.
+Similarly, the values for configurable variables of types array of `map`, array of `record`, and `table` can be now provided using the TOML array of TOML in-line tables.
+
+For example, if the configurable variables are defined in the following way,
+
+```ballerina
+configurable map<anydata> mapVar = ?;
+configurable Person recordVar = ?;
+configurable table<map<int>> tableVar = ?;
+configurable Person[] recordArrayVar = ?;
+
+```
+
+the values can be provided in the `Config.toml` file as follows.
+
+```
+mapVar = {a = "a", b = 2, c = 3.4, d = [1, 2, 3]}
+
+recordVar = {name = "Jane"}
+
+tableVar = [{a = 1, b = 2}, {c = 3}, {d = 4, e = 5, f = 6}]
+
+recordArrayVar = [{name = "Tom"}, {name = "Harry"}]
+
+```
+
+##### Improved Configurable Variables to Support Tuple Types Through TOML Syntax
+
+The configurable feature is improved to support variables of tuple types through the TOML syntax.
+
+For example, if the tuple-typed configurable variables are defined in the following way,
+
+```ballerina
+configurable [int, string, float, decimal, byte, boolean] simpleTuple = ?;
+configurable [int[], [string, int], map<anydata>, table<map<string>>] complexTuple = ?;
+configurable [int, string, int...] restTuple = ?;
+```
+
+the values can be provided in the `Config.toml` file as follows.
+
+```
+simpleTuple = [278, "string", 2.3, 4.5, 2, true]
+
+complexTuple = [[1, 3, 5, 7, 9], ["apple", 2], {name = "Baz Qux", age = 22}, [{a = "a"}, {b = "b", c = "c"}]]
+
+restTuple = [1, "foo", 2, 3, 4, 5]
+```
+
+##### Improved Configurable Variables to Support Union Types Through CLI Arguments
+
+The configurable feature is improved to support variables of union types with simple basic typed members through the CLI arguments.
+
+For example, if the configurable variables are defined in the following way,
+
+```ballerina
+configurable float|int|string unionVar = ?; 
+```
+
+the values can be provided via CLI arguments in the following way.
+
+```
+bal run -- -Cval=5.0
+```
+
+##### Improved Runtime Error Creator and Value Creator API input validations
+
+In order to handle Java Exceptions due to the invalid use of Ballerina runtime error creator and value 
+creator APIs, input validations has been improved to provide proper ballerina runtime errors.
+For example, the following invalid use of `ValueCreator.createRecordValue` API to create a record value with Java ArrayList as a field of it will result in panic.
```

Comment:
```suggestion
For example, the following invalid use of the `ValueCreator.createRecordValue` API to create a record value with a Java ArrayList as a field of it will result in a panic.
```